### ISSUE TITLE
Scala 2.13.3 render issue with 0.12.3

### ISSUE THREAD

**Issue by narma** [pureconfig/pureconfig#782]
Test case 

```scala
import pureconfig.syntax._
import pureconfig._
import pureconfig.generic.auto._

case class Config(configValue: Int = 15)

object Hello extends App {
  val c = Config()
  println(c.toConfig.render())
}
```

scala 2.13.3 + pureconfig 0.12.3:
```
{
    # hardcoded value
    "ymbol-(config-value-)" : 15
}
```

scala 2.13.3 + pureconfig master (builded wit scala 2.13.3):
```
{
    # hardcoded value
    "config-value" : 15
}
```
_

**Comment by jcazevedo** [comment]
Should be fixed with #756. We should be releasing a new version in the meantime with that fix.

**Comment by jcazevedo** [comment]
PureConfig 0.13.0, including the change from #756, was just released.
